app/views/select_transactions.scala.html

```diff
@@ -49,7 +49,7 @@
             form,
             messages("cf.cash-account.transactions.request.from"),
             legendClasses ="govuk-fieldset__legend--s",
-            hintText = Some(messages("cf.cash-account.transactions.request.date.hint-start-date")),
+            hintText = Some(messages("cf.cash-account.transactions.request.date.hint-start-date.v2")),
```

Comment:
Could you please write tests to assert this msg key and msg key at line 62 as well?